tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java

```diff
@@ -196,6 +199,33 @@ public static void runGradleTask(IProject project, String[] tasks, String[] argu
 		launchConfigurationWC.launch(ILaunchManager.RUN_MODE, monitor);
 	}
 
+	public static void runGradleTask(
```

Comment:
This method is very very close to another method in this same class.  can you merge those two methods into one and support both functionality?

Reply:
done